packages/scss/src/components/_nav-side.scss

```diff
@@ -62,7 +63,7 @@
 
 .navSide-link {
 	&.is-active {
-		background: #404040;
+		background: #404040; // :'(
```

Comment:
Indeed, navSide needs a refactor. I'm waiting to have the new version form UX team to start it.